cmd/libs/go2idl/client-gen/testdata/apis/testgroup/v1/register.go

```diff
@@ -41,6 +42,7 @@ func addKnownTypes(scheme *runtime.Scheme) {
 		&v1.DeleteOptions{},
 		&unversioned.Status{},
 		&v1.ExportOptions{})
+	versionedwatch.AddToGroupVersion(scheme, SchemeGroupVersion)
```

Comment:
I don't understand why we need to add it here - it seems to be that your changes doesn't affect client-generation code.